api/.python-version

```diff
@@ -0,0 +1 @@
+3.12.3
```

Comment:
Did you mean to commit this file?

Reply:
Poetry created it I believe.

Reply:
I think it is a pyenv thing. Probably not a bad idea as it should ensure that devs are running Python 3.12.X locally.